src/components/fundraisers/RecentFundraisers/index.js

```diff
@@ -41,13 +41,19 @@ module.exports = React.createClass({
   },
 
   componentWillMount: function() {
-    this.setState({
-      isLoading: true
-    });
+    this.loadPages();
+  },
+
+  loadPages: function() {
+    this.setState({ isLoading: true });
 
     var props = this.props;
 
-    pages.findByCampaign(props.campaignUid, props.type, props.pageSize, props.page, this.onSuccess);
+    if (props.campaignUids) {
+      pages.findByCampaigns(props.campaignUids, props.type, props.pageSize, props.page, this.onSuccess);
```

Comment:
perhaps instead of an if/else, just `pages.findByCampaigns(props.campaignUids || props.campaignUid, ...` ?